spyderlib/spyder.py

```diff
@@ -1144,7 +1146,11 @@ def add_xydoc(text, pathlist):
                                          self.close_dockwidget_action,
                                          self.maximize_action,
                                          None))
+            self.toggle_toolbars_action = create_action(self,
+                                    _("Toggle toolbar visibility"),

```

Comment:
I know it is easier, it was my first option, but then it is weird when having it in a menu with a check box, unless I rename the option when toggling it from 

`Show toolbars` -> `Hide toolbars`

I actually like this option cause it quite explicit always, much like the maximize pane button is updated depending on its status.